### PR TITLE
Add extra FGRM manufacturer_specific codes

### PR DESCRIPTION
**Pull request by cdjackson**
Signed-off-by: Chris Jackson <chris@cd-jackson.com>

### CODE REVIEW

File: bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveCommandClass.java [hunk 1]
```diff
@@ -477,7 +478,8 @@ protected int extractValue(byte[] buffer, int offset, int size) {
         NON_INTEROPERABLE(0xF0, "NON_INTEROPERABLE", null),
 
         // MANUFACTURER_PROPRIETARY class definitions are defined by the manufacturer and device id
-        FIBARO_FGRM_222(0x010F, 0x0301, "FIBARO_FGRM_222", FibaroFGRM222CommandClass.class);
+        FIBARO_FGRM_222(0x010F, 0x0301, "FIBARO_FGRM_222", FibaroFGRM222CommandClass.class),
+        FIBARO_FGRM_222_1(0x010F, 0x0301, "FIBARO_FGRM_222", FibaroFGRM222CommandClass.class);
```
Comment (jongj) [inline bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveCommandClass.java]: Is that correct? Both lines have identical manufacturer and device id?

Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveCommandClass.java]: Oops - the second one should be 302. I got in a muddle with having both OH1 and OH2 source all open in the IDE at the same time :(

Thanks.

**Comment by reyem** [comment]
Any plans to add this to OH2.. I would love to have tilt functionality for my FGRM222 :-)

**Comment by cdjackson** [comment]
Feel free to add it to the database at www.cd-jackson.com if it doesn’t already exist. If it’s in the OH1 database though, then it’s almost certainly in the OH2 database.

**Comment by reyem** [comment]
OK, seems to be added already in OH2. I just checked in code. However, how can I actually define the item in OH2 ?

OH1:
`Rollershutter room_0_jalousie_1_actor_angle {zwave="7:command=MANUFACTURER_PROPRIETARY,type=lamella"}`

In OH2 I just found a dimmer channel
`Rollershutter room_0_jalousie_1_actor_angle {channel="zwave:device:155e646c9e4:node7:switch_dimmer"}` There seems to be no channel available for "lamella"... How can I actually define the command class and type in OH2 in an item definition?

**Comment by cdjackson** [comment]
The Fibaro specific class is not currently available in OH2.

**Comment by reyem** [comment]
OK, understand. Shall I open an issue in OH2 for this, or is there a reason why this code was not migrated?

**Comment by cdjackson** [comment]
It’s not been high on the priority list to work out how best to integrate it as I want to avoid the bodge that we had in OH1.

**Pull request merged**